spec/fixtures/files/expired.lic

```diff
@@ -1,70 +1,3 @@
------BEGIN LICENSE FILE-----
```

Comment:
Moving these files over to Git LFS to reduce repository size as we add more fixture files, e.g. Docker images.

Reply:
Wow, bad idea. Let's not use LFS because the pricing structure just doesn't make sense seeing as CI counts as LFS egress.

Instead, let's reduce file size of any large files under 100MB.